src/gt4sd/frameworks/enzeptional/processing.py

```diff
@@ -1,17 +1,34 @@
 """enzeptional - data processing utilities."""
 
+import random
 from abc import ABC
+from collections import defaultdict
+from itertools import chain
+from operator import methodcaller
 from typing import Generic, List, Optional, Tuple, TypeVar, Union
 
 import numpy as np
 import torch
 from tape.datasets import pad_sequences
 from tape.registry import registry
 from tape.tokenizers import TAPETokenizer
-from transformers import AutoModelWithLMHead, AutoTokenizer
+from transformers import AutoModelForMaskedLM, AutoTokenizer, T5Model, T5Tokenizer
 
 from ..torch import device_claim
 
+
+def merge_dictionaries(lst_Dict):
```

Comment:
`lst_Dict` mixes snake and camel case, for variables and methods we use snake, so `list_of_dictionaries` would be best.
Also add docstring.